caluma/caluma_form/models.py

```diff
@@ -334,6 +334,19 @@ class Document(core_models.UUIDModel):
     )
     meta = models.JSONField(default=dict)
 
+    @property
+    def flat_answer_map(self):
```

Comment:
I'd rather have this as a method, not a `@property`. Just to make clear to the caller that this *does* stuff. Also, because multiple calls to the property will re-evaluate the code.

And a docstring would be nice too

Reply:
done